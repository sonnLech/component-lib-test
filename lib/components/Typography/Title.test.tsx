import { render } from "@testing-library/react";
import { describe, test, expect } from "vitest";
import { Title } from ".";

describe("Title", () => {
  test("should match snapshot", () => {
    const { container } = render(<Title level={1}>Test title</Title>);
    expect(container).toMatchSnapshot();
  });
  test("should render h1", () => {
    const component = render(<Title level={1}>Test title</Title>);
    const title = component.baseElement.querySelector("h1");
    expect(title).toBeDefined();
  });
});
