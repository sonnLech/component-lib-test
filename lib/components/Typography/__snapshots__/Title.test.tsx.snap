// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Title > should match snapshot 1`] = `
<div>
  <h1
    class="text-slate-600 text-2xl font-light"
  >
    Test title
  </h1>
</div>
`;
